seacrowd/sea_datasets/copal/copal.py

```diff
@@ -0,0 +1,155 @@
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@article{wibowo2023copal,
+  title={COPAL-ID: Indonesian Language Reasoning with Local Culture and Nuances},
+  author={Wibowo, Haryo Akbarianto and Fuadi, Erland Hilman and Nityasya, Made Nindyatama and Prasojo, Radityo Eko and Aji, Alham Fikri},
+  journal={arXiv preprint arXiv:2311.01012},
+  year={2023}
+}
+"""
+_DATASETNAME = "COPAL"
+
+_DESCRIPTION = """\
+COPAL is a novel Indonesian language common sense reasoning dataset. Unlike the previous Indonesian COPA dataset (XCOPA-ID), COPAL-ID incorporates Indonesian local and cultural nuances,
+providing a more natural portrayal of day-to-day causal reasoning within the Indonesian cultural sphere.
+Professionally written by natives from scratch, COPAL-ID is more fluent and free from awkward phrases, unlike the translated XCOPA-ID.
+Additionally, COPAL-ID is presented in both standard Indonesian and Jakartan Indonesian–a commonly used dialect.
+It consists of premise, choice1, choice2, question, and label, similar to the COPA dataset.
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/haryoaw/COPAL"
+
+_LICENSE = Licenses.CC_BY_SA_4_0.value
+
+_URLS = {
```

Comment:
I think this is less intuitive, can we keep this as single-level dict? just remove the `_DATASETNAME`, wdyt?

Reply:
agree lol. done